.github/workflows/static_build.yml

```diff
@@ -159,7 +159,7 @@ jobs:
             nlohmann_json
             simdjson-static>=3.3.0
             spdlog
-            fmt>=11.1.0
+            fmt>=11.2.0
```

Comment:
If we decide to not support previous versions, we should remove [this function](https://github.com/mamba-org/mamba/blob/main/libmambapy/src/libmambapy/bindings/utils.cpp#L25-L32) and use the correct api directly where needed (`!is_terminal_color()`).